cfn-resources/maintenance-window/cmd/resource/resource.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
-	mongodbatlas "go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/atlas-sdk/v20230201008/admin"
```

Comment:
I know we have mixed opinions on using an alias, would like to get consensus moving forward so we are consistent across all resources (we can agree in daily stand up)

Reply:
I agree, I feel like we should use the name the Go packaged decided (except if naming conflict) but I agree we should have a common approach